packages/shared/utils/storage.ts

```diff
@@ -1,8 +1,26 @@
-import { ClusterServiceVersionKind } from '@odf/shared/types';
+import { OCSStorageClusterModel } from '@odf/shared/models';
+import {
+  ClusterServiceVersionKind,
+  StorageSystemKind,
+} from '@odf/shared/types';
 import { ODF_OPERATOR } from '../constants';
 
 export const getODFCsv = (csvList: ClusterServiceVersionKind[] = []) =>
   csvList.find(
     (csv) =>
       csv?.metadata.name?.substring(0, ODF_OPERATOR.length) === ODF_OPERATOR
   );
+
+export const getOCSStorageSystem = (ssList: StorageSystemKind[] = []) =>
+  ssList.find(
+    (ss) =>
+      ss?.spec?.kind ===
+      getGVKLabel({
+        kind: OCSStorageClusterModel.kind,
+        apiVersion: OCSStorageClusterModel.apiVersion,
+        apiGroup: OCSStorageClusterModel.apiGroup,
+      })
```

Comment:
```suggestion
      getGVKLabel(OCSStorageClusterModel);
```
this should work as well, right ?
similar to how we have done above...

Reply:
no, it won't, OCSStorageClusterModel has apiGroup optional. but getGVKLabel has apiGroup non-optional.  we need to make apiGroup args optional on getGVKLabel to work but it can't be optional here

Reply:
OCSStorageClusterModel is a static object, we know that apiGroup will always be present in it...
we can pass like this: getGVKLabel(`OCSStorageClusterModel as any`) instead of adding three extra lines everytime we are calling getGVKLabel fucntion.
wdyt ??

Reply:
or we can handle type in a better way...

Reply:
I feel its ok, this might be the safe way. If you see here model also created on same way: https://github.com/red-hat-storage/odf-console/blob/master/packages/odf/components/create-storage-system/external-storage/index.ts#L18